src/main/java/games/strategy/engine/data/GameParseException.java

```diff
@@ -1,6 +1,8 @@
 package games.strategy.engine.data;
 
-public class GameParseException extends Exception {
+import org.xml.sax.SAXException;
+
+public class GameParseException extends SAXException {
```

Comment:
Inheriting from `SAXException` is a leaky abstraction.  Clients of `GameParser` shouldn't know or care that we're using a SAX, DOM, XPP, etc. parser.  `SAXException`s thrown internally to the parser should be wrapped in a `GameParseException` and rethrown.